### PR TITLE
Ruby 2.5

### PR DESCRIPTION
**Pull request by wesrich**
* Remove Debian-based `Dockerfile`
* Rename `Dockerfile.ubuntu` to `Dockerfile`
* Update Ruby version to `2.5`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,98 +1,76 @@
...
 LABEL maintainer="kevin.olbrich@mckesson.com"
 
-ENV NODE_VERSION 8.11.3
-ENV YARN_VERSION 1.7.0
-ENV PHANTOMJS_VERSION 2.1.1
-ENV CHEFDK_VERSION 1.6.11
-ENV DOCKERIZE_VERSION v0.6.0
+# Java
 ENV JAVA_HOME=/jdk1.6.0_45
 ENV PATH="${JAVA_HOME}/bin:${PATH}"
```
Comment (basicBrogrammer) [inline Dockerfile]: Is this building from the JAVA downloaded onto MacOS ? ??? 😱

File: Dockerfile [hunk 2]
```diff
@@ -1,98 +1,76 @@
...
-  && curl -fSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz" \
-  && curl -fSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz.asc" \
-  && gpg --batch --verify yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz \
-  && mkdir -p /opt \
-  && tar -xzf yarn-v$YARN_VERSION.tar.gz -C /opt/ \
-  && ln -s /opt/yarn-v$YARN_VERSION/bin/yarn /usr/local/bin/yarn \
-  && ln -s /opt/yarn-v$YARN_VERSION/bin/yarnpkg /usr/local/bin/yarnpkg \
-  && rm yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz
+# Dockerize
+ENV DOCKERIZE_VERSION v0.6.0
```
Comment (basicBrogrammer) [inline Dockerfile]:
```
dockerize is a utility to simplify running applications in docker containers. It allows you to:

generate application configuration files at container startup time from templates and container environment variables
Tail multiple log files to stdout and/or stderr
Wait for other services to be available using TCP, HTTP(S), unix before starting the main process.
``` 


So docker-compose ?

File: Dockerfile [hunk 3]
```diff
@@ -1,98 +1,76 @@
...
+RUN curl --compressed -L --output dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+  && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+  && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
 
 # PhantomJS
-RUN curl --output /usr/local/bin/phantomjs https://s3.amazonaws.com/circle-downloads/phantomjs-$PHANTOMJS_VERSION \
+ENV PHANTOMJS_VERSION 2.1.1
```
Comment (basicBrogrammer) [inline Dockerfile]: when are we going to kill the `deprecated phantom` ? #soapbox

File: Dockerfile [hunk 4]
```diff
@@ -1,98 +1,76 @@
...
+RUN curl --compressed -L --output /usr/local/bin/phantomjs https://s3.amazonaws.com/circle-downloads/phantomjs-$PHANTOMJS_VERSION \
   && chmod a+x /usr/local/bin/phantomjs
 
-# CodeClimate
-RUN curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > /usr/local/bin/cc-test-reporter \
-  && chmod a+x /usr/local/bin/cc-test-reporter
+# Ruby
+ENV RUBY_VERSION 2.3
```
Comment (basicBrogrammer) [inline Dockerfile]: thought we were moving to ruby 2.5 ?

File: Dockerfile [hunk 5]
```diff
@@ -1,98 +1,76 @@
-FROM ruby:2.3.7-stretch
+FROM ubuntu:16.04
```
[the remark this answers is not in the document]
Comment (johncalvinyoung) [reply Dockerfile]: Did not change.

File: Dockerfile (hunk 3 above)
Comment (johncalvinyoung) [reply Dockerfile]: Not yet. You want to volunteer to stabilize all the Chrome-fragile tests? :trollface:

File: Dockerfile (hunk 1 above)
Comment (wesrich) [reply Dockerfile]: Yes, it's a file in the repo. There's no other way than downloading the installer from Oracle and installing manually.

File: Dockerfile (hunk 4 above)
Comment (johncalvinyoung) [reply Dockerfile]: arggggggggh good catch. I reverted it by accident.

File: Dockerfile (hunk 3 above)
Comment (wesrich) [reply Dockerfile]: When someone takes the time to update all of our projects to not use it? 😉

File: Dockerfile (hunk 2 above)
Comment (wesrich) [reply Dockerfile]: I don't know. We're using it to ping Postgres to ensure it starts up before moving on to a next step. But it doesn't really let you run commands elsewhere ☹️

File: Dockerfile (hunk 5 above)
[the remark this answers is not in the document]
Comment (wesrich) [reply Dockerfile]: Not an option on AWS yet.

fullscreen_7_16_18__3_16_pm

[the remark this answers is not in the document]
Comment (basicBrogrammer) [reply Dockerfile]: 👍

**Pull request merged**